### ISSUE TITLE
Pass project-id-version as parameter

### ISSUE THREAD

**Issue by colindean** [xitrum-framework/scala-xgettext#15]
I'd like to "version" the pot files by giving it the project name and the commit hash at which the i18n.pot file was generated. Those aren't really details that xgettext should extract itself, so perhaps let the build system pass that as an option.

**Comment by ngocdaothanh** [comment]
I think setting project-id-version in the pot file doesn't matter much, because:

The normal workflow is like this:

Create language files for the first time:
* Generate English pot file: i18n.pot
* Copy it to create various languages: fr.po, de.po, cs.po etc. Then translate those files.
* If you want, you can set project-id-version to those files.

Regularly, when strings in the program source code are updated, you want to update the above files:
* Regenerate the English pot file: i18n.pot
* Use tools like Poedit to automatically merge i18n.pot to fr.po, de.po, cs.po etc. that have already been translated.
* If project-id-version is in i18n.pot, that may or may not be what you want to overwrite to the po files. This may bring headache.

Another headache is that the po files are part of a bigger thing, like a program or a jar file. That bigger thing is usually already versioned. Setting project-id-version may cause the problem that versions in po files are not in sync with version of the bigger thing (inconsistent).

In short, setting project-id-version may bring much headache.

**Comment by colindean** [comment]
Should that header then be removed from the default header, or commented out, if it is not required to be set?

**Comment by ngocdaothanh** [comment]
If it doesn't cause problem, I think just leave it.
